src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -9,40 +9,52 @@ docs_area: manage
 {% include_cached feature-phases/limited-access.md %}
 {{site.data.alerts.end}}
 
-[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) autoprovisioning centralizes and automates provisioning and deprovisioning of {{ site.data.products.db }} organization users and groups from your identity provider (IdP).
+[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) autoprovisioning centralizes and automates provisioning and deprovisioning for {{ site.data.products.db }} organization users and groups from your identity provider (IdP).
```

Comment:
I would include "identity management" somewhere here, since in addition to provisioning and deprovisioning it centralizes and automates user management (e.g. updates to user profile information)